compiler/rustc_error_codes/src/error_codes/E0798.md

```diff
@@ -0,0 +1,38 @@
+Functions marked as `C-cmse-nonsecure-call` place restrictions on their
+inputs and outputs.
+
+- inputs must fit in the 4 available 32-bit argument registers. Alignment
+is relevant.
+- outputs must either fit in 4 bytes, or be a foundational type of
+size 8 (`i64`, `u64`, `f64`).
+
+For more information,
+see [arm's aapcs32](https://github.com/ARM-software/abi-aa/releases).
+
+Erroneous code example:
+
+```compile_fail,E0798
+#![feature(abi_c_cmse_nonsecure_call)]
+
+#[no_mangle]
+pub fn test(
+    f: extern "C-cmse-nonsecure-call" fn(u32, u32, u32, u32, u32) -> u32,
+) -> u32 {
+    f(1, 2, 3, 4, 5)
+}
+```
```

Comment:
CI fails for the code samples in this file. I suspect something weird is going on because this snippet only gives the asserted error with a very specific target, and the test run on these files does not take the target into consideration.

@GuillaumeGomez are there similar examples among the other error codes? I'm not getting a clear error message and therefore not sure what to do here.

Reply:
Not that I know of.

Reply:
so, would it be ok to skip the "```compile_fail,E0798" bit here? then this doesn't really test anything anymore but that seems like it's the best I can do right now

Reply:
If it doesn't work on all target, I think it's ok.